lib/crypto/eosdart/src/serialize.dart

```diff
@@ -416,7 +416,7 @@ class SerialBuffer {
   String getPrivateKey() {
     var type = get();
     var data = getUint8List(numeric.privateKeyDataSize);
-    return numeric.privateKeyToString(numeric.IKey(type as numeric.KeyType, data));
+    return numeric.privateKeyToString(numeric.IKey(numeric.KeyType.values[type], data));
```

Comment:
What does this do and why does this fix things?

Reply:
IIRC this was a suggestion from the linter. It's just a way to cast the serialized byte representing keytype into the right Dart type. No interesting semantics.